HW2/P5/driver.py

```diff
@@ -5,19 +5,40 @@
 import set_compiler
 set_compiler.install()
 
+import pstats, cProfile
+
 import pyximport
 pyximport.install()
-
+import pdb
 import numpy as np
 from timer import Timer
 from animator import Animator
 from physics import update, preallocate_locks
 
+###############################
+#Morton encoding function pulled from https://en.wikipedia.org/wiki/Z-order_curve
+def cmp_zorder(a, b):
```

Comment:
Did you check that this actually worked correctly, i.e. visualize the sorting? I tried the wikipedia one and it only worked for small numbers, i.e. for matrices less than 2^16 or so.